tools/build_browser.js

```diff
@@ -169,9 +169,11 @@ function installDemoStyles() {
 
   glob.sync("**", { cwd: "./src/styles" }).forEach((file) => {
     const stat = fss.statSync(`./src/styles/${file}`);
+    if (stat.isDirectory()) return;
+
     if (file.endsWith(".css")) {
-      installCleanCSS(`./src/styles/${file}`, `demo/styles/${file}`);
-    } else if (!stat.isDirectory) {
+      installCleanCSS(`./src/styles/${file}`, `demo/styles/${file}`, false);
```

Comment:
Isn't this a change in behavior?  Those files are currently minified and this would make them not, right?

Reply:
Data change yes, functional no. I have choosed non-minified style for demo styles.

Reply:
> Non-minified style for demo styles is probably preferred

_Oh... I wasn't even thinking about the demo site..._What is the thinking that non-minified should be preferred?  I don't think anyone is looking at these files - and [if they are] don't modern browser make it easy enough to review CSS in play regardless of whether the source files are minified or not?

Reply:
Without source map they don't show the source nicely.

Reply:
Let's focus on the CDN and Node.js builds as originally discussed.  Every modern browser I'm aware of allows you to (with built-in dev tools) easily not only edit the CSS live - but also un-minifying it as well to easily read it.  **I'm not seeing any need to change the Demo site.**  If someone wants the original files they can always fetch the repository from GitHub itself and start from there.

Reply:
Done.